### PR TITLE
Add cmake build for benchmark_model

### PR DESCRIPTION
**Pull request by snnn**

### CODE REVIEW

File: tensorflow/tools/benchmark/benchmark_model.cc [hunk 1]
```diff
@@ -272,7 +272,11 @@ Status TimeMultipleRuns(double sleep_seconds, int num_runs,
     // This can be helpful to determine the effect of mobile processor
     // scaling and thermal throttling.
     if (sleep_seconds > 0.0) {
+#ifdef _WIN32
```
Comment (petewarden) [inline tensorflow/tools/benchmark/benchmark_model.cc]: Minor nit, but can you add an include to [tensorflow/core/platform/platform.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/platform/platform.h) and use the PLATFORM_WINDOWS macro instead? That is a bit more idiomatic.

**Review by petewarden — COMMENTED** [review]
Thanks, this is looking great overall! Just one minor comment.

File: tensorflow/tools/benchmark/benchmark_model.cc (hunk 1 above)
Comment (snnn) [inline tensorflow/tools/benchmark/benchmark_model.cc]: ok.

Comment (snnn) [inline tensorflow/tools/benchmark/benchmark_model.cc]: Done.

**Review by petewarden — APPROVED** [review]
Thanks for the update, looks good!

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Pull request merged**